exchange.go

```diff
@@ -492,7 +493,7 @@ func (re *RabbitExchangeImpl) Receive(exchange ExchangeSettings, queue QueueSett
 		}, nil
 }
 
-func (re *RabbitExchangeImpl) BulkReceive(exchange ExchangeSettings, queue QueueSettings, maxWait time.Duration) (func(BulkMessageHandleFunc) error, func(), error) {
+func (re *RabbitExchangeImpl) BulkReceive(exchange ExchangeSettings, queue QueueSettings, maxWait time.Duration, batchSize int) (func(BulkMessageHandleFunc) error, func(), error) {
```

Comment:
Only one recommendation: Maybe add a default batch size, in case someone forgets to specify a non-zero batch size?